cloud-functions/functions/firebase-repl.js

```diff
@@ -13,6 +13,9 @@ const firebaseConfig = require(`./config/firebase.config.${process.env.NODE_ENV}
 firebase.initializeApp(firebaseConfig);
 admin.initializeApp(firebaseConfig);
 
+// initialize firebase analytics
+firebase.analytics();
```

Comment:
This was a repl script I built a long time ago, and is only ever called if we call it explicitly from the command line, so this change likely isn't doing what you think it is.